### ISSUE TITLE
如何在1.8中定制实现ExponentialLR

### ISSUE THREAD

**Issue by allarobot** [PaddlePaddle/Paddle#27148]
你好，aistudio中目前只支持到paddle1.8，目前还没有develop版本中才出现的lr_scheduler，请问如何在1.8中自定义一些lr scheduler,譬如cosineAnnealing,Warmup, ExponentialLR等呢？

**Comment by mapingshuo** [comment]
这个API满足不了你的需求吗：https://paddlepaddle.org.cn/documentation/docs/zh/api_cn/layers_cn/exponential_decay_cn.html#exponential-decay

**Comment by allarobot** [comment]
谢谢，原来在layer下面有这个。
如果我希望是一个自定义的方法或者类customized_scheduler，譬如达到这个效果：
if decay_steps>N: 
       learning_rate = learning_rate * decay_rate ^ (decay_steps)
else:
       learning_rate = learning_rate

定义optimizer时，这么用
import paddle.fluid as fluid
base_lr = 0.1
sgd_optimizer = fluid.optimizer.SGD(
    learning_rate=**customized_scheduler**(
        learning_rate=base_lr,
        decay_steps=10000,
        decay_rate=0.5,
        staircase=True))

可以实现吗，或者有其它推荐的实现方法吗？

**Comment by allarobot** [comment]
@mapingshuo 
关于自定义scheduler，能给我一点建议吗？
